rad/src/types/array.rs

```diff
@@ -14,13 +14,13 @@ use crate::{
     operators::{array as array_operators, identity, Operable, RadonOpCodes},
     script::RadonCall,
     types::{
-        float::RadonFloat, map::RadonMap, mixed::RadonMixed, string::RadonString, RadonType,
-        RadonTypes,
+        boolean::RadonBoolean, bytes::RadonBytes, float::RadonFloat, integer::RadonInteger,
+        map::RadonMap, string::RadonString, RadonType, RadonTypes,
     },
 };
 
 fn bytes_discriminant() -> Discriminant<RadonTypes> {
-    discriminant(&RadonTypes::from(RadonMixed::from(Value::Null)))
+    discriminant(&RadonTypes::Fail)
```

Comment:
Im using this empty type to tag MixedArray case. What do you think about this solution @aesedepece and @tmpolaczyk?